### PR TITLE
Sort items in namespace drop down by name

### PR DESCRIPTION
**Pull request by SaschaSchwarze0**
# Changes

The longer list of items in the Namespace drop down is sometimes driving me crazy to find what I look for. Changing it to show items sorted to make that easier.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/main/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/main/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/main/standards.md#commit-messages)

### CODE REVIEW

**Comment by AlanGreene** [comment]
Thanks for the PR @SaschaSchwarze0. Can you share more details about the environment you're seeing this in? I'm seeing the namespaces displayed in sorted order without this change, see for example our dogfooding cluster: https://dashboard.dogfooding.tekton.dev/#/pipelines

**Comment by SaschaSchwarze0** [comment]
> Thanks for the PR @SaschaSchwarze0. Can you share more details about the environment you're seeing this in? I'm seeing the namespaces displayed in sorted order without this change, see for example our dogfooding cluster: https://dashboard.dogfooding.tekton.dev/#/pipelines

Hi @AlanGreene, sure, I am using microk8s. Here is parts of my drop down:

grafik

It is kind of sorted in the order the namespaces were created, I think.

File: src/containers/NamespacesDropdown/NamespacesDropdown.test.js [hunk 1]
```diff
@@ -41,8 +41,9 @@ it('NamespacesDropdown renders items', () => {
   namespaces.forEach(item => {
     expect(queryByText(new RegExp(item, 'i'))).toBeTruthy();
   });
-  getAllByText(/namespace-/i).forEach(node => {
-    expect(namespaces.includes(getNodeText(node))).toBeTruthy();
+  const sortedNamespaces = namespaces.sort();
```
Comment (AlanGreene) [inline src/containers/NamespacesDropdown/NamespacesDropdown.test.js:44]: This is an in-place sort so `namespaces` will be re-ordered here too

**Review by AlanGreene — APPROVED** [review]
I haven't been able to reproduce the issue in my test environments but the change is safe and solves a problem that may affect other users.

Just one potential issue with the test, otherwise this looks good 👍 

/ok-to-test

File: src/containers/NamespacesDropdown/NamespacesDropdown.test.js (hunk 1 above)
Comment (SaschaSchwarze0) [reply src/containers/NamespacesDropdown/NamespacesDropdown.test.js]: Indeed, good catch.

**Comment by AlanGreene** [comment]
Thanks again for the PR
/lgtm

**Pull request merged**